src/main/webapp/script.js

```diff
@@ -60,6 +60,54 @@ function checkPermissions() {
   });
 }
 
+function hideAccessForm() {
+  this.style.display = 'none';
+  document.getElementById('content').style.display = 'block';
+}
+
+function makeFormDissapear() {
+  const access = document.getElementById('access');
+
+  // Code for Chrome, Safari and Opera
+  access.addEventListener("webkitAnimationEnd", hideAccessForm);
+
+  // Standard syntax
+  access.addEventListener("animationend", hideAccessForm);
+
+  access.style.WebkitAnimation = "fade 0.5s 1"; // Code for Chrome, Safari and Opera
+  access.style.animation = "fade 0.5s 1";
+}
+
+function accessFunction() {
+  const projectId = document.forms['accessForm']['projectIdAccess'].value;
+  const bucketName = document.forms['accessForm']['bucketName'].value;
+  const objectName = document.forms['accessForm']['objectName'].value;
+  
+  if (projectId.length === 0) {
+    document.getElementById('projectIdAccess').style.borderColor = 'red';
+    return;
+  }
+
+  if (bucketName.length === 0) {
+    document.getElementById('bucketName').style.borderColor = 'red';
+    return;
+  }
+
+  if (objectName.length === 0) {
+    document.getElementById('objectName').style.borderColor = 'red';
+    return;
+  }
+
+  makeFormDissapear();
```

Comment:
same here

Reply:
Solved in https://github.com/googleinterns/step229-2020/pull/70/commits/1d3f009d4f51b391b23a70e256daec9e669af1f4